app/src/main/res/values-fr-rFR/strings.xml

```diff
@@ -0,0 +1,147 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+  <string name="app_name" translatable="false">Noice</string>
```

Comment:
```suggestion
```

Reply:
Ok for the translatable=false, I'm currently applying your diffs in one big commit for more consistency

Reply:
You can just add these suggestions to batch and apply them in a single commit i think.

Reply:
oh okay, I didn't know how to do that so I just did it from the command line, I just pushed so I will it the "resolve" button